AzureMonitor/Scripts/Trigger-PolicyInitiativeRemediation.ps1

```diff
@@ -290,8 +297,9 @@ if($ManagementGroup)
     $SubScriptionsToProcess =@()
     if($ManagementGroupID)
     {
+        $azEnvironment = Get-AzEnvironment -Name $Environment
         $GetBody = BuildBody -method "GET"
-        $MGSubsDetailsURI = "https://management.azure.com/providers/microsoft.management/managementGroups/$($ManagementGroupID)/descendants?api-version=2018-03-01-preview"
```

Comment:
This currently doesn't appear to be working properly. See comment in Create-AzDiagPolicy.PS1  Needs to be

$MGSubsDetailsURI = "$($azEnvironment.ResourceManagerUrl)providers/microsoft.management/managementGroups/$($ManagementGroupID)/descendants?api-version=2018-03-01-preview"

Reply:
Done in 
ebe972d